bin/condor_submit

```diff
@@ -63,21 +64,101 @@ def main():
         - get credentials
         - handle tarfile options
         - set added values from environment, etc.
-        - convert/render template files to submission files
         - launch
     """
-    cmd_args = sys.argv[1:]
 
-    proxy, token = get_creds(args)
+    condor_parser = argparse.ArgumentParser(
+        description='Queue jobs for execution under HTCondor without the jobsub wrapper.')
+
+    condor_parser.add_argument(
+        "-G",
+        "--group",
+        required=False,
+        help="Group/Experiment/Subgroup for priorities and accounting",
+        action=StoreGroupinEnvironment)
+
+    condor_parser.add_argument(
+        "-name",
+        required=False,
+        help="Submit to the specified condor_schedd")
+
+    condor_parser.add_argument(
+        "-spool",
+        required=False,
+        help="Spool all required input files, job event log, and proxy over the connection to the condor_schedd",
+        action='store_true')
+
+    condor_parser.add_argument(
+        "-remote",
+        required=False,
+        help="Submit to the specified condor_schedd, spooling all required input files over the network connection")
+
+    condor_parser.add_argument(
+        "--credentials",
+        help="credential information")
```

Comment:
Currently this doesn't support anything, right?  Do you think it's best to comment this out for now, or perhaps add a note saying that it's not implemented yet?

Reply:
Correct. I will comment that it is not implemented.